index.ts

```diff
@@ -1,3 +1,10 @@
+type MapLike<K, V> = {
```

Comment:
👋🏾 from Web Systems - bumping this comment https://github.com/github/memoize/pull/25#issuecomment-2277988913